### ISSUE TITLE
The fuzz test cases defined in Bytes._fuzz_library[] are never used

### ISSUE THREAD

**Issue by withdk** [jtpereyda/boofuzz#466]
**Brief Description
============**
Based on the code comments, the Bytes primitive self._fuzz_library[] "contains binary strings [which] will always included as testcases" .

Unfortunately, this variable gets clobbered after the Bytes class is initialised and super() is called. This occurs because  _Bytes._fuzz_library_ is defined before super() is called, and the parent _BasePrimitive.fuzz_library_ overwrites it.

**Affected Code
===========**
```
./primitives/bytes.py:
    # This binary strings will always included as testcases.
    _fuzz_library = [
        b"",
        b"\x00",
        b"\xFF",
        b"A" * 10,
        b"A" * 100,
        b"A" * 1000,
        b"A" * 5000,
        b"A" * 10000,
        b"A" * 100000,
    ]

...snip...
def __init__(self, value, size=None, padding=b"\x00", fuzzable=True, max_len=None, name=None):
...snip...
super(Bytes, self).__init__() ; <<<< This clobbers self._fuzz_library 
```

**Quick fix
=======**
The fix I used was to simply rename Bytes._fuzz_library. Another option would be to setup self._fuzz_library after the super() is called.

**Comment by SR4ven** [comment]
Thanks for reporting @withdk!

We're currently reworking the whole test case generation system in #422. The bug you're describing here might be affected or even fixed there.
If not, we'll work out a fix. Thanks for providing possible solutions.

**Comment by keisentraut** [comment]
This is my fault, I implemented it wrong in #302. However, the current API for mutators was cumbersome anyway which is the root cause of bugs like this one. Since #422 is finally merged, I will take another look at s_bytes (and maybe s_string) again.

**Comment by SR4ven** [comment]
Fixed by #422
